### PR TITLE
940 create more detailed log output

### PR DESCRIPTION
**Pull request by laurenzhonauer**
### Checklist

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/openkfw/TruBudget/blob/master/CONTRIBUTING.md).
- [x] I fixed all necessary PR warnings
- [x] The commit history is clean
- [x] The E2E tests are passing
- [x] If possible, the issue has been divided into more subtasks
- [x] I did a self review before requesting a review from another team member

### Description

Closes ModifyMe

### CODE REVIEW

File: blockchain/src/connectToChain.js [hunk 1]
```diff
@@ -49,15 +52,15 @@ function startSlave(
     },
     "Start slave nodes with params",
   );
-  const args = [
+  const args = getDeamonArguments([
```
[the remark this answers is not in the document]
Comment (laurenzhonauer) [reply blockchain/src/connectToChain.js:55]: Done

File: email-notification-service/src/middleware.ts [hunk 2]
```diff
@@ -30,11 +31,13 @@ export const verifyUserJWT = (req: Request, res, next, secret: string) => {
       next();
     })
     .catch((err) => {
+      logger.trace("User-JWT invalid", token);
```
[the remark this answers is not in the document]
Comment (laurenzhonauer) [reply email-notification-service/src/middleware.ts:34]: Done

File: email-notification-service/src/middleware.ts [hunk 3]
```diff
@@ -43,6 +46,7 @@ export const verifyNotificationJWT = (req: Request, res, next, secret: string) =
       next();
     })
     .catch((err) => {
+      logger.trace("Notification-JWT invalid", token);
```
[the remark this answers is not in the document]
Comment (laurenzhonauer) [reply email-notification-service/src/middleware.ts:49]: Done

File: email-notification-service/src/sendMail.ts [hunk 4]
```diff
@@ -3,15 +3,17 @@ import config from "./config";
...
     host: config.smtpServer.host,
     port: config.smtpServer.port,
     secure: config.smtpServer.secure,
     auth: {
       user: config.smtpServer.user,
       pass: config.smtpServer.password,
     },
-  });
+  };
+  logger.trace("Sending email with following configuration: ", transportOptions);
```
[the remark this answers is not in the document]
Comment (laurenzhonauer) [reply email-notification-service/src/sendMail.ts:15]: Done

File: excel-export-service/src/api.ts [hunk 5]
```diff
@@ -106,9 +110,11 @@ export async function getSubprojects(
   token: string,
   base: string,
 ): Promise<Subproject[]> {
+  logger.trace("Fetching subprojects ...");
   const response: AxiosResponse<SubprojectResponse> = await axios
     .get(`${base}/subproject.list?projectId=${projectId}`, getAuthHeader(token))
     .catch(function (error) {
+      logger.error(error);
```
[the remark this answers is not in the document]
Comment (laurenzhonauer) [reply excel-export-service/src/api.ts:117]: Done

File: excel-export-service/src/index.ts [hunk 6]
```diff
@@ -96,23 +95,25 @@ excelService.get("/download", async (req: express.Request, res: express.Response
...
   }
 });
 
-excelService.listen(config.serverPort, () => log.info(`App listening on ${config.serverPort}`));
+excelService.listen(config.serverPort, () => logger.info(`App listening on ${config.serverPort}`));
 
 // Enable useful traces of unhandled-promise warnings:
 process.on("unhandledRejection", (err) => {
-  log.error({ err }, "UNHANDLED PROMISE REJECTION");
+  logger.error({ err }, "UNHANDLED PROMISE REJECTION");
```
[the remark this answers is not in the document]
Comment (laurenzhonauer) [reply excel-export-service/src/index.ts:111]: Done

File: excel-export-service/src/api.ts [hunk 7]
```diff
@@ -122,19 +128,22 @@ export async function getWorkflowitems(
...
   base: string,
 ): Promise<Workflowitem[]> {
+  logger.trace("Fetching workflowitems ...");
   const response: AxiosResponse<WorkflowitemResponse> = await axios
     .get(
       `${base}/workflowitem.list?projectId=${projectId}&subprojectId=${subprojectId}`,
       getAuthHeader(token),
     )
     .catch(function (error) {
+      logger.error(error);
```
[the remark this answers is not in the document]
Comment (laurenzhonauer) [reply excel-export-service/src/api.ts:138]: Done

**Review by mayrmartin — APPROVED** [review]

**Pull request merged**